api/src/main/java/io/druid/js/JavaScriptConfig.java

```diff
@@ -35,16 +35,14 @@
   private static final JavaScriptConfig ENABLED_INSTANCE = new JavaScriptConfig(true);
 
   @JsonProperty
-  private boolean enabled = false;
+  private final boolean enabled;
 
   @JsonCreator
   public JavaScriptConfig(
       @JsonProperty("enabled") Boolean enabled
   )
   {
-    if (enabled != null) {
-      this.enabled = enabled.booleanValue();
-    }
+    this.enabled = enabled != null && enabled.booleanValue();
```

Comment:
This is kind of hurting my head. Maybe I'm just bad at mental boolean logic.

Since the default value of `enabled` is false, it would be simpler to change the `Boolean` to `boolean` in the constructor parameter, and then just write `this.enabled = enabled;`. (the `boolean` type will default to false if not provided in json)